satellite/accounting/live/redis.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package live
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"go.uber.org/zap"
+
+	"storj.io/storj/storage"
+	"storj.io/storj/storage/redis"
```

Comment:
Avoid direct dependencies to a database implementation in satellite services etc. The reason is to make it easy to add new database implementation, without requiring it to be a dependency to a specific binary. This probably is less important to satellite than to storage node and uplink, but still a good practice.

Reply:
I think I see what you mean. Pass a storage.KeyValueStore into the service? But what should I do with the `plainMemoryLiveAccounting` that was already there, since it doesn't match the KeyValueStore interface?

Reply:
You would pass a specific `Service` or `DB` rather than `storage.KeyValueStore`... The key value is one possible implementation, not the definitive.